etna/core/blocks/quote.py

```diff
@@ -17,3 +17,26 @@ class Meta:
         icon = "openquote"
         label = "Quote"
         template = "blocks/quote.html"
+
+
+class ReviewBlock(QuoteBlock):
+    """
+    Gives an editor the ability to add a review from an entity, e.g.
+    a newspaper, that has given us a review. A quote block with stars.
+    """
+
+    stars = blocks.ChoiceBlock(
```

Comment:
At the moment, this is coming out in the API as a string. Could it be an integer instead?